### DISCUSSION TITLE
Missing required parameter for [Route: language-switcher.locale]

### DISCUSSION THREAD

**Discussion by blondie63** [Laravel-Backpack/community-forum#899] [Q&A (Help)] [answered]
I updated all my projects to Laravel 11 without problems, but on one I have this error using the language switcher which I also use on all the others without problems, what could it be?

<img width="1402" alt="BKP6-LV11-LangSelectorIssue" src="https://github.com/Laravel-Backpack/community-forum/assets/1261851/0c2faba8-d931-49e3-babe-21226a9c1241">

Thanks for help

**Comment by pxpm** [comment]
Hey @blondie63 

I think the issue there is that you are using the switcher in a "non-auth" page (login .. before user is logged in), and you are including the middleware only in the routes where user is logged in. 

I think this is what you are missing: https://github.com/Laravel-Backpack/language-switcher?tab=readme-ov-file#can-i-use-this-package-outside-of-the-backpackadmin-scope

Let me know how it goes.

Cheers

**Reply by blondie63** [reply, marked as answer]
Hi @pxpm i've just enabled this middleware, i fixed it restarting vs.code.. very strange..
